### ISSUE TITLE
Install fails for yanked mimemagic 0.3.5 dependency

### ISSUE THREAD

**Issue by bjoernmartin** [avalonmediasystem/avalon-docker#58]
At some point of the Docker installation (during `docker compose up` step) the following error is shown and the build / deployment terminates:
```
Fetching source index from https://rubygems.org/
Your bundle is locked to mimemagic (0.3.5), but that version could not be found
in any of the sources listed in your Gemfile. If you haven't changed sources,
that means the author of mimemagic (0.3.5) has removed it. You'll need to update
your bundle to a version other than mimemagic (0.3.5) that hasn't been removed
in order to install.
ERROR: Service 'avalon' failed to build: The command '/bin/sh -c bundle install --with aws development test postgres --without production' returned a non-zero code: 7
```
According to https://stackoverflow.com/a/66921259/6499594 this is because the `mimemagic` version `0.3.5` was yanked (whatever that means).

As a result the current Docker installation 7.2 cannot be deployed.

**Comment by apertome** [comment]
We're aware of this issue, and it has been fixed in some images. Which docker images are you trying to use?

This is not directly related to #4440 but can be worked on simultaneously. 

If you are trying to build the image, you can fix this issue locally by issuing this command and then try rebuilding the image:

```
bundle update mimemagic
```

Thanks,
Michael

**Comment by bjoernmartin** [comment]
I just followed the instructions at https://github.com/avalonmediasystem/avalon-docker#usage, so it's whatever version is specified within https://github.com/avalonmediasystem/avalon-docker/blob/master/docker-compose.yml at the moment. That would be `avalonmediasystem/avalon:7.2`. 

I then tried building my own image. First I executed your suggestion from above, `bundle update mimemagic`, which required me to install a bit of ruby / gem stuff. It failed occasionally due to a gem missing which I had to install explicitly. When it came to installing SQLite that path didn't work anymore and I was stuck.

As a last resort I just executed `docker build -t myorg/avalon:version --target=prod .`, as suggested at https://github.com/avalonmediasystem/avalon#docker-deployment, without updating `mimemagic`. But that failed immediately with the initial `mimemagic` issue.

As I'm no ruby developer I'll probably just wait for the next release and retry then.

But thanks for the quick reply!

**Comment by apertome** [comment]
Hi again @bjoernmartin , we are working on a bugfix release to address this and a couple other issues. In the meantime, if you want to test the new docker image, you can change the docker image that's specified to  `avalonmediasystem/avalon:7.2.1` -- it's not a final release yet but so far we have not seen any issues with it.

**Comment by bjoernmartin** [comment]
Thanks for providing a preview version, @apertome , much appreciated! 

I finally got around trying it, and I got farther than before. Unfortunately after quite some time I get this message:
```
avalon_1  | bash: rails: command not found
avalon-docker_avalon_1 exited with code 127
```
To make sure there is nothing _left over_ from my last try with version 7.2, I pruned all Docker containers and removed all Docker images, hopefully forcing a complete reload + rebuild. But the outcome is the same. Here's the output of `docker logs avalon-docker_avalon_1` after the second try:
```
2021-05-02T12:26:01.446Z 7 TID-gp9yprbur INFO: Cron Jobs - add job with name: Scan for batches - every 1min
2021-05-02T12:26:01.456Z 7 TID-gp9yprbur INFO: Cron Jobs - add job with name: Status Checking and Email Notification of Existing Batches - every 15min
2021-05-02T12:26:01.467Z 7 TID-gp9yprbur INFO: Cron Jobs - add job with name: Status Checking and Email Notification for Stalled Batches - every 1day
2021-05-02T12:26:01.477Z 7 TID-gp9yprbur INFO: Cron Jobs - add job with name: Clean out user sessions older than 7 days - every 6hour
bash: rails: command not found
```
The other containers continue to run and I see periodical log output from `solr_1`, `worker_1` and `redis_1`. 

I tried connecting to http://localhost/ anyway since the nginx container came up as well, but that does not work. Instead, I can reach nginx at http://localhost:8880/. But there is no Avalon showing up, just the standard _Welcome to nginx!_ message is served.

I hope this helps in any way.

**Comment by apertome** [comment]
Hi @bjoernmartin,

We have released 7.2.1 which fixes the mimemagic issue, as well as the `rails` error. So make sure you pull down the latest copy of `avalon-docker` github repo as well as the docker image. Hopefully that will fix it. Let me know if that works for you.

Thanks!

**Comment by bjoernmartin** [comment]
Hi @apertome . The install just finished and I now have a working installation of Avalon.

Thanks a lot for your help!
